src/components/app.jsx

```diff
@@ -32,10 +28,11 @@ export default class App extends React.Component {
     const session = enigma.create(config);
     try {
       qixGlobal = await session.open();
-      const appHandle = await qixGlobal.getDoc(); // Mixin from ./src/enigma/get-doc
-      const appLayout = await appHandle.getAppLayout();
-      appHandle.on('changed', this.appChanged);
-      this.setState({ session, app: appHandle, lastReloadTime: appLayout.qLastReloadTime });
+      const app = await qixGlobal.getDoc();
+      app.on('changed', this.appChanged);
+      this.setState({ session, app });
+      // trigger initial fetch:
+      app.emit('changed');
```

Comment:
Ah. That's a nice trick :)

Reply:
Yes, as mentioned though, if you have unknown listeners to the model events it might not be the best idea since you'll trigger `changed` for all of them :)